Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -2216,8 +2252,11 @@ def long_running_execution_command(client: Client, params: dict):
     mirror_options = params.get('mirror_options', DEFAULT_MIRRORING_DIRECTION)
     mirror_direction = MIRROR_DIRECTION.get(mirror_options)
     mirror_options = params.get('mirror_options', '')
+    assets_limit = int(params.get('assets_limit', DEFAULT_ASSETS_LIMIT))
     if not argToBoolean(params.get('retry_events_fetch', True)):
         EVENTS_SEARCH_TRIES = 1
+    is_reset_triggered()
+    recover_from_last_run()
```

Comment:
please add both of these function an option to get context as argument so you can get it one time and then pass it to both.
under engine it cold be a big difference.

Reply:
fixed